lib/registries/addon/components/registries-navbar/styles.scss

```diff
@@ -168,3 +168,14 @@
         font-size: 20px;
     }
 }
+
+:global(.brand-container) {
```

Comment:
This is meant to only style the `.Logo` class when a brand was successfully fetched. I made a change to the `with-branding` modifier so that it will add a `brand-container` class. Notice that it will do an early return if a brand was not fetched.